pdns/auth-catalogzone.hh

```diff
@@ -41,7 +41,7 @@ public:
     Consumer
   };
 
-  static const char* getTypeString(enum CatalogType type)
+  static constexpr const char* getTypeString(enum CatalogType type)
   {
     const char* types[] = {"none", "producer", "consumer"};
```

Comment:
this could be a static const char as well.

Reply:
Look like it's `--std=c++2b` aka C++23.
```
../../pdns/auth-catalogzone.hh: In static member function 'static constexpr const char* CatalogInfo::getTypeString(CatalogType)':
../../pdns/auth-catalogzone.hh:46:24: error: 'types' defined 'static' in 'constexpr' function only available with '-std=c++2b' or '-std=gnu++2b'
   46 |     static const char* types[] = {"none", "producer", "consumer"};
      |                        ^~~~~
In file included from ../../pdns/dnsbackend.hh:46,
                 from ../../pdns/ueberbackend.hh:33,
                 from ../../pdns/dnsseckeeper.hh:34,
                 from bindbackend2.cc:40:
../../pdns/auth-catalogzone.hh: In static member function 'static constexpr const char* CatalogInfo::getTypeString(CatalogType)':
../../pdns/auth-catalogzone.hh:46:24: error: 'types' defined 'static' in 'constexpr' function only available with '-std=c++2b' or '-std=gnu++2b'
   46 |     static const char* types[] = {"none", "producer", "consumer"};
      |                        ^~~~~
```